### PR TITLE
MVP-24640: Request to Create OpenAPI Spec doc for RedlineAPIv2 

### PR DESCRIPTION
**Pull request by clawyq**
Added api spec for redlining api (JSON/YML)

### CODE REVIEW

**Review by arizonasingh — APPROVED** [review]
LGTM. Left one suggestion.

qq: isn't this a brand new api? confused why this is under v2 and v2 is in the request uri.

File: openapi/schemas/v2/Markup/GetRedlines.json [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    "info": {
+      "version": "2.0",
+      "title": "Item Attachment Put",
+      "license": {
+        "name": "MIT"
+      }
+    },
+    "servers": [
+      {
+        "url": "https://{orgId}.salesforce.com/services/apexrest/{namespace}/v2",
```
Comment (arizonasingh) [inline openapi/schemas/v2/Markup/GetRedlines.json]: highly recommend endpoint > orgId. most people don't know their org id off the top of their head, whereas base endpoint is a bit more intuitive. orgId is also a misnomer as I assume you want org instance (e.g. NA159) not the actual org id, though I could be wrong.
```suggestion
        "url": "https://{endpoint}/services/apexrest/{namespace}/v2",
```

File: openapi/schemas/v2/Markup/GetRedlines.yml [hunk 2]
```diff
@@ -0,0 +1,112 @@
+---
+openapi: 3.0.0
+info:
+  version: '2.0'
+  title: Item Attachment Put
+  license:
+    name: MIT
+servers:
+- url: https://{orgId}.salesforce.com/services/apexrest/{namespace}/v2
```
Comment (arizonasingh) [inline openapi/schemas/v2/Markup/GetRedlines.yml]: highly recommend endpoint > orgId. most people don't know their org id off the top of their head, whereas base endpoint is a bit more intuitive. orgId is also a misnomer as I assume you want org instance (e.g. NA159) not the actual org id, though I could be wrong.
```suggestion
- url: https://{endpoint}/services/apexrest/{namespace}/v2
```

**Comment by clawyq** [comment]
> LGTM. Left one suggestion.
> 
> qq: isn't this a brand new api? confused why this is under v2 and v2 is in the request uri.

The redline API is built on top of the `v2` path when Ron first started working on PDX 8 months ago. The mapping in our source code is `@RestResource(urlMapping='/api/v2/item/*/markup')`

File: openapi/schemas/v2/Markup/GetRedlines.json (hunk 1 above)
Comment (clawyq) [reply openapi/schemas/v2/Markup/GetRedlines.json]: Good point. Seeing that I based this spec off of `bom-get.json`, I'm going to do a sweep of the entire repo and do this replacement.

**Review by arizonasingh — APPROVED** [review]

**Pull request merged**